typescript/sdk/src/providers/MultiProtocolProvider.ts

```diff
@@ -99,12 +100,12 @@ export class MultiProtocolProvider<
     const providers = objMap(
       this.providers,
       (_, typeToProviders) => typeToProviders[ProviderType.EthersV5]?.provider,
-    ) as ChainMap<EthersV5Provider['provider'] | undefined>;
+    ) as ChainMap<ZKSyncProvider['provider'] | undefined>;
```

Comment:
Similar to the concern I expressed above around the Wallet class, I disagree that ZKSyncProviders should replace Ethers V5 providers as the default provider type here

Reply:
You are right, this should not be here. Resolved